src/mixins/browserCheck.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @copyright Copyright (c) 2019 Marco Ambrosini <marcoambrosini@pm.me>
+ *
+ * @author Marco Ambrosini <marcoambrosini@pm.me>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { showError } from '@nextcloud/dialogs'
+
+const browserCheck = {
+	methods: {
+		checkBrowser() {
+			if (!this.isFullySupported) {
+				showError(
+					this.unsupportedWarning,
+					{
+						timeout: 3600,
+						isHTML: true,
+					})
+			}
+		},
+	},
+	computed: {
+		isFullySupported() {
+			return (this.$browserDetect.isFirefox && this.$browserDetect.meta.version >= 52)
+			|| (this.$browserDetect.isChrome && this.$browserDetect.meta.version >= 49)
+		},
+		// Disable the call button and show the tooltip
+		blockCalls() {
+			return (this.$browserDetect.isFirefox && this.$browserDetect.meta.version < 52)
+			|| (this.$browserDetect.isChrome && this.$browserDetect.meta.version < 49)
```

Comment:
```suggestion
			return (this.$browserDetect.isFirefox && this.$browserDetect.meta.version < 52)
			|| (this.$browserDetect.isChrome && this.$browserDetect.meta.version < 49)
			|| this.$browserDetect.isIE
```
you forgot the most important part